modules/services/window-managers/i3-sway/lib/functions.nix

```diff
@@ -102,6 +102,18 @@ rec {
             optionalString (colors.separator != null)
             "separator ${colors.separator}"
           }
+          ${
+            optionalString (colors.focusedBackground != null)
+            "focused_background ${colors.focusedBackground}"
```

Comment:
the newlines this generates in tests is annoying, can't we refactor this to avoid the newlines ?

Reply:
Something like
``` nix
concatStringsSep "\n" (
  [ ]
  ++ optional (cfg.something) "string"
  ++ optional .....)
```

Reply:
We have https://github.com/nix-community/home-manager/issues/2030 to track this